apps/web/src/pages/api/forms/editStudy.ts

```diff
@@ -22,7 +22,7 @@ export default withApiHandler<ExtendedNextApiRequest>(Roles.SponsorContact, asyn
     assert(CPMS_API_USERNAME)
     assert(CPMS_API_PASSWORD)
 
-    const { studyId, plannedClosureDate, plannedOpeningDate, actualClosureDate, actualOpeningDate, ...studyData } =
+    const { id, studyId, plannedClosureDate, plannedOpeningDate, actualClosureDate, actualOpeningDate, ...studyData } =
```

Comment:
could we call the studyId in here cpmsId (and maybe the Id StudyId) - a bit confusing

Reply:
updated, made the naming more clear, let me know what you think